source/base/quadrature.cc

```diff
@@ -449,108 +456,6 @@ namespace internal
   }   // namespace QIteratedImplementation
 } // namespace internal
 
-// template <>
```

Comment:
huh

Reply:
This function does almost the same thing as the constructor which is why we got rid of it. Are you asking for a change?